src/sass/layout/_boxmodel.scss

```diff
@@ -15,6 +15,10 @@
     padding: .75em;
 }
 
+.fluff-none {
+  padding: 0;
```

Comment:
@zachmonteith don't add a single zeroed value for a utility class. Just removing `fluff-tight--full` would accomplish the same thing. In the end, we are removing this entire bar for now, so I'm closing this PR – just wanted to give you some feedback on your change.